Sources/ArgumentParser/DialogBuilder/Check.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift Argument Parser open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+/// Ask the user to confirm the prompt.
+///
+/// ```swift
+/// guard
+///   check("Are you sure you want to delete all?")
+/// else { return }
+/// print("---DELETE---")
+/// ```
+///
+/// The above code will generate the following dialog:
+///
+/// ```
+/// Are you sure you want to delete all?
+/// ? Please enter [y]es or [n]o: y
+/// ---DELETE---
+/// ```
+///
+/// - Parameter prompt: The prompt to display.
+/// - Returns: The user decision.
+public func check(_ prompt: String) -> Bool {
```

Comment:
It can be used to correct misspelled arguments, such as:

```
% example --indx 5
Error: Unexpected argument '--indx', did you mean '--index'?
? Please enter [y]es or [n]o: y
```

But this feature is optional, and it doesn't look like we have enough time to implement it before the deadline.